staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -44,6 +44,7 @@ type Interface interface {
 	Post() *Request
 	Put() *Request
 	Patch(pt types.PatchType) *Request
+	Apply() *Request
```

Comment:
it has to have the same signature here: `Apply(force bool, manager string) *Request`

Reply:
oh right I will add that now. My computer just isnt liking this repo